### ISSUE TITLE
[FEAT]: Add preset to the Calibrate, Simulate and Optimize operator

### ISSUE THREAD

**Issue by YohannParis** [DARPA-ASKEM/terarium#4201]
### Design
![image](https://github.com/user-attachments/assets/bea596c5-c9b7-43a7-8c7e-07a4a3281a0a)

### Branch dev
https://github.com/DARPA-ASKEM/terarium/compare/main...TS/3433-speed-quality-button

### value for the dropdown
To be filled by @liunelson

**Comment by liunelson** [comment]
Preset = `Fast` (i.e. `Debug, Test` mode)
* Number of samples = 1
* Solver method = "euler"
* Number of global steps (`maxiter`) = 0
* Number of local steps (`maxfeval`) = 1

Preset = `Precise` (i.e. `Serious, Evaluation, ?` mode)
* Number of samples = 100
* Solver method = "dopri5"
* Number of global steps (`maxiter`) = 
* Number of local steps (`maxfeval`) =

**Comment by liunelson** [comment]
There is some minor issues with the design of Calibrate:
* The fields called "Number of samples" and "Solver method" stay as they are
* There should be no "Number of global steps, number of local steps" field  in the Settings section of Calibrate (those are for Optimize _only_)
* Instead, there should be the following:
1. A locked dropdown field called "Inference algorithm" with default value `SVI`
2. A locked dropdown field called "Loss function" with default value `ELBO` (paired with "Inference algorithm")
3. A locked dropdown field called "Optimizer" with default value `Adam`
4. A numeric field called "Learning rate" with default value 0.03 (paired with "Optimizer")

Preset = `Fast` (i.e. `Debug, Test, ?` mode)
* Number of samples = 1
* Solver method = "euler"
* Number of iterations = 10
* Learning rate = 0.1

Preset = `Precise` (i.e. `Serious, Evaluation, ?` mode)
* Number of samples = 100
* Solver method = "dopri5"
* Number of iterations = 1000
* Learning rate = 0.03

**Comment by liunelson** [comment]
For Optimize, here are some defaults:

## Preset = Fast (i.e. Debug, Test, ? mode)

Number of samples to evaluate constraints = 10
Number of samples to simulate model = 1
ODE solver method = "euler"
Number of global steps (maxiter) = 0
Number of local steps (maxfevals) = 1

## Preset = Precise (i.e. Serious, Evaluation, ? mode)

Number of samples to evaluate constraints = 100
Number of samples to simulate model = 100
ODE solver method = "dopri5"
Number of global steps (`maxiter`) = 5
Number of local steps (`maxfeval`) = 25

**Comment by dvince2** [comment]
Reopening for optimize
